Libplanet.Tools/Mpt.cs

```diff
@@ -0,0 +1,183 @@
+                {
+                    Console.Error.Write("At ");
+                    Console.WriteLine(ByteUtil.Hex(pair.Root.ToByteArray()));
+                    Console.WriteLine(pair.Value.Inspection);
+                }
+
+                Console.WriteLine();
+            }
+        }
+
+        // FIXME: Now, it works like `set` not `add`. It allows override.
+        [Command(Description="Add a new mpt store alias.")]
+        public void Add(
+            [Argument] string alias,
+            [Argument] string uri,
+            [FromService] IConfigurationService<string, string> configurationService)
+        {
+            try
+            {
+                // Checks the `uri` is valid.
+                ParseKVStoreURI(uri);
+            }
+            catch (Exception e)
+            {
+                throw new CommandExitedException(
+                    $"It seems the uri is not valid. (exceptionMessage: {e.Message})", -1);
+            }
+
+            configurationService.Set(MPTKVStoreAliasKey(alias), uri);
+        }
+
+        [Command(Description="Remove the registered mpt store.")]
+        public void Remove(
+            [Argument] string alias,
+            [FromService] IConfigurationService<string, string> configurationService)
+        {
+            configurationService.Delete(MPTKVStoreAliasKey(alias));
+        }
+
+        [PrimaryCommand]
+        public void Help([FromService] ICoconaHelpMessageBuilder helpMessageBuilder)
+        {
+            Console.Error.WriteLine(helpMessageBuilder.BuildAndRenderForCurrentContext());
+        }
+
+        private (KVStoreType KVStoreType, SchemeType SchemeType, string Path) ParseKVStoreURI(
+            string rawUri)
+        {
+            var uri = new Uri(rawUri);
+            var scheme = uri.Scheme;
+            var splitScheme = scheme.Split("+");
+            if (splitScheme.Length <= 0 || splitScheme.Length > 2)
+            {
+                var exceptionMessage = "The scheme must be explicit and must be formatted with " +
+                                       "two methods, '<kv-store-type>' or " +
+                                       "'<kv-store-type>+<uri-scheme>'.";
+                throw new ArgumentException(exceptionMessage, nameof(rawUri));
+            }
+
+            SchemeType schemeType = SchemeType.File;
+            if (!KVStoreType.TryParse(
+                splitScheme[0], ignoreCase: true, out KVStoreType kvStoreType))
+            {
+                throw new NotSupportedException(
+                    $"There is no supported kv-store-type for '{splitScheme[0]}'");
+            }
+
+            if (splitScheme.Length == 2
+                && !SchemeType.TryParse(splitScheme[1], ignoreCase: true, out schemeType))
+            {
+                throw new NotSupportedException(
+                    $"There is no supported scheme type for '{splitScheme[1]}'");
+            }
+
+            return (kvStoreType, schemeType, uri.AbsolutePath);
```

Comment:
Why do we need `KVStoreType` thing and `LoadKeyValueStore()` method?  Can't we make this `ParseKVStoreURI()` method to directory return `IKeyValueStore` instead of `KVStoreType`?

Reply:
I understood your comment as that let us merge a function to parse and another function to load kv-store into one. Then the method `ParseKVStoreURI()` will be return a single value typed `IKeyValueStore` not tuple, right?

Reply:
Yes, that's what I meant.